cross-functional/workflow.md

```diff
@@ -1,34 +1,125 @@
+- **Task lists**: Are specific actions or deliverables that are taken to complete an initiative. These are defined as `[tasklist]` objects in each initiative, and are the main units of work that drive our actions.
+
+````{admonition} Example structure of an initiative
+:class: dropdown
+Here's the rough structure of an initiative, use this to guide the creation of new initiatives.
+
+```markdown
+title: Initiative name in GitHub issue
+
+### Description
+
+A short description of the initiative, with enough context so that people can understand
+what kind of work it will entail, and why it is important to 2i2c's overall goals.
+
+### Linked goal or project
+
+- Link to a strategic goal or a project (2i2c-based, upstream, etc)
+
+### Why is this important?
+
+- This will allow us to XXX
+
+### Measures and definition of done
 
-The [`#solution-forum` Slack channel](https://2i2c.slack.com/archives/C06G5FAAT63) is a space for cross-organization initiatives and strategic conversation.
+- We expect to see XXX
+- This will be complete when YYY
 
-- It is meant for discussing organization-wide practices, policies, and planning.
-- It will have a combination of strategic- and tactical-level discussion. (in the future we may split this into two channels if it's too much)
-- It will initially focus its discussion around our "cross-functional initiatives" board and the items in it.
-- Anyone that participates in conversations in this channel should dedicate the attention to understand the context for discussion and participate until topics are resolved.
-- Area leads are expected to monitor this channel and participate in cross-team conversations as they arise.
-- All 2i2c staff have visibility into the channel and may participate in conversations, but are not expected to do so as part of their daily responsibilities.
+```[tasklist]
+- [ ] A growing list of tasks and actions used to coordinate work around this initiative.
+- [ ] Each should be an actionable issue that we track independently.
+- [ ] This may be empty when the initiative is created! The initiative's goals should be used to decide what action issues to create next.
+```
+````
 
-### Team members expected to monitor this channel
+### Bi-weekly initiatives meetings
 
-Below is a list of the team members that we **expect** to participate in this channel on a regular basis:
+There are two bi-weekly meetings with the initiatives team.
 
+- **Initiative planning and refinement**: A combindation of **coordination** and **prioritization**. We first walk the board and ensure each initiative has clear next steps, assigned people, and that any coordination between teams is clear. We next zoom out and discuss our overall portfolio of initiatives to ensure we're sequencing them properly.
+- **Initiative retrospectives**: The Initiatives team follows the same retrospective structure we follow for any team.
```

Comment:
yes! I agree that it is really helpful to follow the 'structure' we follow for any team. It means we can be more efficient about the our very valuable synchronous meeting time.